Fix missing import in registration API

femtobit opened the pull request:
While using a local Indico installation (version 2.1.2), I noticed that check-in did not work with the Indico Check-In Android app from Google Play Store. This was due to a `NameError` being thrown in line 61 here:
https://github.com/indico/indico/blob/9fa826ebb672021b59ecc2744e98ad1eaaabb4b8/indico/modules/events/registration/api.py#L59-L61

Importing the `signals` model from `indico.core` fixes this.

mic4ael commented:
I am curious how it got into the master. However this is a fix that should be merged to the branch `2-1.maintenance`

ThiefMaster commented:
Indeed, if you could rebase this PR onto `2.1-maintenance` and change the upstream branch of your PR as well that would be great. Since this justifies a 2.1.3 release, I'll take the opportunity during the next few days to see which other small fixes/improvements we have in master that might be worth backporting now that there will be a 2.1.3 after all.

In any case, thanks for spotting and fixing this!

femtobit commented:
> In any case, thanks for spotting and fixing this!

You're welcome. Thanks for the quick response.

As requested, I rebased unto `2.1-maintenance` and changed the base branch of this PR accordingly.

ThiefMaster commented:
Thanks! We'll include this in a 2.1.3 bugfix release.
